scripts/write_cmpd_to_frag_dict.py

```diff
@@ -0,0 +1,69 @@
+"""
+The goals is to create a saved dictionary mapping the compound_ID to the fragalysis structure we are using.
+i.e. `AAR-POS-0daf6b7e-1: Mpro-x1311`
+The input is the compound_tracker.csv file, the output is a yaml file (default is in data/cmpd_to_frag.yaml).
+This generates a required input for the `fauxalysis_from_docking.py` script.
+Example Usage:
+    python write_cmpd_to_frag_dict.py
+        -f ~/Scientific_Projects/mers-drug-discovery/Mpro-paper-ligand/extra_files/Mpro_compound_tracker_csv.csv
+
+"""
+import argparse, os, sys, yaml
+
+repo_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(repo_path)
+
+from covid_moonshot_ml.datasets.utils import (
+    get_compound_id_xtal_dicts,
+    parse_fragalysis_data,
+)
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description="")
+
+    parser.add_argument(
+        "-f",
+        "--frag_csv",
```

Comment:
Not a blocker. I would just call it `input` here and adapt the arguments as needed. The help should say whatever format it expects or if it's related to fragalysis, etc.

Reply:
since this script is specific to generating the `COMPOUND_ID: FRAGALYSIS_DATASET` dictionary from the fragalysis csv file I this it makes sense, esp since we're using `parse_fragalysis_data` here to process the csv